stock_picking_invoicing/models/stock_picking_type.py

```diff
@@ -0,0 +1,31 @@
+# Copyright (C) 2019-Today: Odoo Community Association
+# @ 2019-Today: Akretion - www.akretion.com.br -
+#   Magno Costa <magno.costa@akretion.com.br>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models, fields
+
+
+class PickingType(models.Model):
+    _inherit = "stock.picking.type"
+
+    count_picking_2binvoiced = fields.Integer(
+        compute='_compute_picking_2binvoiced')
+
+    def _compute_picking_2binvoiced(self):
+        domains = {
+            'count_picking_2binvoiced': [('invoice_state', '=', '2binvoiced')],
+        }
+        for field in domains:
+            data = self.env[
+                'stock.picking'].read_group(
+                domains[field] + [
+                    ('state', 'not in', ('done', 'cancel')),
```

Comment:
This domain must be only:
```
('state', '!=', 'cancel'),
```
The user must see the pickings that are already done, and ready to invoice; Or may be the 2 pick too, but 2binvoiced;

And the view domain must be the same, because you se for example 3 in the dashboard, and when you open you have 10.